aiqc/__init__.py

```diff
@@ -288,106 +288,135 @@ def update_config(kv:dict):
+			# get_config() will print a null condition.
+			pass
 		else:
-			print(
-				f"=> Yikes - failed to create tables.\n" \
-				f"Please see README file section titled: 'Deleting & Recreating the Database'\n"
-			)
+			db_path = aiqc_config['db_path']
+
+		self.db_path = db_path
+
+
+	def get_path_db(self):
+		"""
+		Originally, this code was in a child directory.
+		"""
+		aiqc_config = get_config()
+		if aiqc_config is None:
+			# get_config() will print a null condition.
+			pass
+		else:
+			db_path = aiqc_config['db_path']
+			return db_path
 
 
-def destroy_db(confirm:bool=False, rebuild:bool=False):
-	if (confirm==True):
-		db_path = get_path_db()
-		db_exists = os.path.exists(db_path)
+	def get_db(self):
+		"""
+		The `BaseModel` of the ORM calls this function.
+		"""
+		if self.db_path is None:
+			print("\n=> Info - Cannot fetch database yet because it has not been configured.\n")
+		else:
+			db = SqliteExtDatabase(self.db_path)
+			return db
+
+
+	def create_db(self):
+		# Future: Could let the user specify their own db name, for import tutorials. Could check if passed as an argument to create_config?
+		db_exists = os.path.exists(self.db_path)
 		if db_exists:
+			print(f"\n=> Skipping database file creation as a database file already exists at path:\n{self.db_path}\n")
+		else:
+			# Create sqlite file for db.
 			try:
-				os.remove(db_path)
+				db = self.get_db()
 			except:
 				print(
-					f"=> Yikes - failed to delete database file at path:\n{db_path}\n\n" \
+					f"=> Yikes - failed to create database file at path:\n{self.db_path}\n\n" \
 					f"===================================\n"
 				)
 				raise
-			print(f"\n=> Success - deleted database file at path:\n{db_path}\n")
+			print(f"\n=> Success - created database file at path:\n{self.db_path}\n")
+
+		db = self.get_db()
+		# Create tables inside db.
+		tables = db.get_tables()
+		table_count = len(tables)
+		if (table_count > 0):
+			print(f"\n=> Info - skipping table creation as the following tables already exist.{tables}\n")
 		else:
-			print(f"\n=> Info - there is no file to delete at path:\n{db_path}\n")
-		importlib.reload(sys.modules[__name__])
+			db.create_tables([
+				File, Tabular, Image,
+				Dataset,
+				Label, Feature,
+				Splitset, Featureset, Foldset, Fold,
+				Encoderset, Labelcoder, Featurecoder,
+				Algorithm, Hyperparamset, Hyperparamcombo,
+				Queue, Jobset, Job, Predictor, Prediction,
+				FittedEncoderset, FittedLabelcoder,
+				Window
+			])
+			tables = db.get_tables()
+			table_count = len(tables)
+			if table_count > 0:
+				print("\n💾  Success - created all database tables.  💾\n")
+			else:
+				print(
+					f"=> Yikes - failed to create tables.\n" \
+					f"Please see README file section titled: 'Deleting & Recreating the Database'\n"
+				)
 
-		if (rebuild==True):
-			create_db()
-	else:
-		print("\n=> Info - skipping destruction because `confirm` arg not set to boolean `True`.\n")
+
+	def new_db(self, db_name:str):
```

Comment:
It feels like `new_db` should just be a part of `create_db`

so we would have a single method called `Database.create(name:str=<some_default that can be overriden>)`